test/cpp/api/functional.cpp

```diff
@@ -380,3 +380,32 @@ TEST_F(FunctionalTest, PReLU) {
   const auto y_exp = (x < 0) * w * x  + (x >= 0) * x;
   ASSERT_TRUE(torch::allclose(y, y_exp));
 }
+
+TEST_F(FunctionalTest, ReLU) {
+  const auto size = 3;
+  for (const auto inplace : {false, true}) {
+    auto x = torch::linspace(-10.0, 10.0, size * size * size);
+    x.resize_({size, size, size});
+    auto y_exp = (x < 0) * 0 + (x >= 0) * x;
+    auto y = F::relu(x, ReLUOptions().inplace(inplace));
```

Comment:
It would be great to add a test case for `F::relu(x, /*inplace=*/inplace)` as well :D